src/ocean/meta/codegen/CTFE.d

```diff
@@ -0,0 +1,94 @@
+/*******************************************************************************
+
+    Utilities for common CTFE operations needed for code generation
+
+    Copyright:
+        Copyright (c) 2017 sociomantic Labs GmbH. All rights reserved.
+
+    License:
+        Boost Software License Version 1.0. See LICENSE_BOOST.txt for details.
+        Alternatively, this file may be distributed under the terms of the Tango
+        3-Clause BSD License (see LICENSE_BSD.txt for details).
+
+*******************************************************************************/
+
+module ocean.meta.codegen.CTFE;
+
+import ocean.meta.types.Qualifiers;
+
+/*******************************************************************************
+
+    Converts integer to string
+
+    Params:
+        i = integer to convert
+
+*******************************************************************************/
+
+istring toString ( long i )
```

Comment:
We already have https://github.com/sociomantic-tsunami/ocean/blob/bcd88397f484ca4821ceed942b518f05c2e2274f/src/ocean/core/Traits_tango.d#L707-L805 . If you want to move it there, those should be deprecated. Also, why only the signed version is provided ?